migrations/017.do.cancel-stuck-transactions.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE transactions_pending (
+  hash TEXT NOT NULL PRIMARY KEY,
+  timestamp TEXT NOT NULL,
```

Comment:
```suggestion
  timestamp TIMESTAMPTZ NOT NULL,
```

Since our goal is to have more type safety.